### PR TITLE
Bugfix: no support for XCPng 8.2.1

### PR DESCRIPTION
**Pull request by nvazquez**
### Description

This PR adds initial support for XCPng 8.2.1 for a minimal working version based on the workaround found on #6349 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?

### CODE REVIEW

**Comment by nvazquez** [comment]
@blueorangutan package

File: engine/schema/src/main/resources/META-INF/db/schema-41610to41700.sql [hunk 1]
```diff
@@ -933,3 +933,11 @@ CREATE VIEW `cloud`.`event_view` AS
         `cloud`.`projects` ON projects.project_account_id = event.account_id
             LEFT JOIN
         `cloud`.`event` eve ON event.start_id = eve.id;
+
+-- Add XenServer 8.2.1 hypervisor capabilities
+INSERT IGNORE INTO `cloud`.`hypervisor_capabilities`(uuid, hypervisor_type, hypervisor_version, max_guests_limit, max_data_volumes_limit, max_hosts_per_cluster, storage_motion_supported) VALUES (UUID(), 'XenServer', '8.2.1', 1000, 253, 64, 1);
+
+-- Copy XenServer 8.2.0 hypervisor guest OS mappings to XenServer 8.2.1
+INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) SELECT UUID(),'Xenserver', '8.2.1', guest_os_name, guest_os_id, utc_timestamp(), 0 FROM `cloud`.`guest_os_hypervisor` WHERE hypervisor_type='Xenserver' AND hypervisor_version='8.2.0';
+INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (hypervisor_type, guest_os_name, guest_os_id, hypervisor_version, uuid, created, removed, is_user_defined) VALUES ('Xenserver', 'Debian Bullseye 11', 337, '8.2.1', uuid(), now(), NULL, 0);
```
Comment (weizhouapache) [inline engine/schema/src/main/resources/META-INF/db/schema-41610to41700.sql:942]: @nvazquez 
would be good to use the procedure `cloud`.`ADD_GUEST_OS_AND_HYPERVISOR_MAPPING` ?

**Comment by weizhouapache** [comment]
this seems not to be a bug :-D

**Comment by nvazquez** [comment]
Ping for review @AlexanderKgr @benoitlair

**Review by rohityadavcloud — APPROVED** [review]

File: engine/schema/src/main/resources/META-INF/db/schema-41610to41700.sql (hunk 1 above)
Comment (nvazquez) [reply engine/schema/src/main/resources/META-INF/db/schema-41610to41700.sql]: Done, thanks

**Comment by nvazquez** [comment]
@blueorangutan package

**Review by weizhouapache — APPROVED** [review]
code lgtm

**Comment by nvazquez** [comment]
Verified on vanilla env, no need to run tests

**Comment by AlexanderKgr** [comment]
please add also this

INSERT IGNORE INTO cloud.hypervisor_capabilities(uuid, hypervisor_type,
hypervisor_version, max_guests_limit, max_data_volumes_limit,
max_hosts_per_cluster, storage_motion_supported) values (UUID(), 'XenServer',
'8.2.1', 1000, 253, 64, 1);

**Pull request merged**